rspotify-model/src/show.rs

```diff
@@ -35,7 +37,6 @@ pub struct SimplifiedShow {
     pub publisher: String,
     #[serde(rename = "type")]
     pub _type: String,
```

Comment:
Should we remove the `_type` field?

Reply:
Ahh the thing is that `SimplifiedShow` is undocumented, so we don't know what `_type` is exactly. I will leave it as it is to avoid breakage and add a note.

Reply:
It probably works just like `FullShow`, in which case it's correct to do so. So, done.